docs/requirements.txt

```diff
@@ -6,10 +6,14 @@ h5py
 tqdm
 nbsphinx
 IPython
-docutils <0.17
-sphinx == 5.3.0
```

Comment:
Have you checked that this works without the version specification?

Reply:
It does, but I should add on the new version spec for what I'm using now to prevent things breaking in the future.